src/app/containers/StakePage/components/CurrentVests.tsx

```diff
@@ -4,9 +4,14 @@ import { translations } from 'locales/i18n';
 import { StyledTable } from './StyledTable';
 import { VestingContract } from './VestingContract';
 import { useListOfUserVestings } from '../../../components/UserAssets/Vesting/useListOfUserVestings';
+import { VestGroup } from 'app/components/UserAssets/Vesting/types';
 
 interface Props {
```

Comment:
Man, you know I am sorry but I'll need you to rename the interface and rewrite the component to `React.FC` :D